### DISCUSSION TITLE
Ближайшие планируемые обновления

### DISCUSSION THREAD

**Discussion by link1107** [link1107/DonationExecutor#18] [Объявления и анонсы]
- Веб-интерфейс для удобного создания конфиг-файла
- Расширение для twitch с выезжающим списком цен
- Возможность добавления кастомных гигантских мобов через конфиг-файл
- Добавление поддержки twitch-bits

**Comment by gmankab** [comment]
Игорь, вы лучший. Спасибо Вам большое ❤️❤️❤️
